oauth2client/contrib/appengine.py

```diff
@@ -477,15 +469,15 @@ def _parse_state_value(state, user):
         user: google.appengine.api.users.User, The current user.
 
     Raises:
-        InvalidXsrfTokenError: if the XSRF token is invalid.
+        ValueError: if the XSRF token is invalid.
```

Comment:
`ValueError` indicates that someone made a programming mistake (either the authors of the library that raises the `ValueError` or the users of the library that raises the `ValueError`, depending on context). If the exception being raised here does not indicate a programming defect, we should use some exception other than `ValueError`. This is a non-public method so what is raised from it needn't be a public exception type, but it's starting to sound like it should be a custom exception type.